docs/json/radarr/cf/hdr10plus-boost.json

```diff
@@ -20,7 +20,7 @@
       "negate": true,
       "required": true,
       "fields": {
-        "value": "^(?=.*\\b(HDR(10)?)\\b)(?=.*\\b(DV|DoVi)\\b)"
+        "value": "^(?=.*\\b(HDR(10)?(?!\\+))\\b)(?=.*\\b(DV|DoVi)\\b)"
```

Comment:
why is this needed?

"name": "Not HDR10", covers this

Reply:
Just did a search and replace for the current DV HDR10 regex and replaced it in all files